jobs/web/spec

```diff
@@ -117,6 +117,12 @@ properties:
       The value to set for X-Frame-Options.
     default: deny
 
+  concurrent_request_limits:
+    env: CONCOURSE_CONCURRENT_REQUEST_LIMIT
+    description: |
+      Limit the number of concurrent requests to an API endpoint.
+    example: ListAllJobs:5
```

Comment:
This can actually be a map, just like the flag type:

```yaml
example:
  ListAllJobs: 5
```

The templating magic will figure out the right format.